### DISCUSSION TITLE
Is there a way to emulate the behavior of Latex's \vfill?

### DISCUSSION THREAD

**Discussion by cristobaltapia** [typst/typst#1066] [General]
Hi all, I was playing with typst the other day (having lots of fun :D ) and was trying to recreate one of my Latex templates. The only thing that I did not manage to do was to emulate the behavior of the Latex command `\vfill` that fills the rest of the space in the page and puts the content at the very end (bottom) of the page. What would you use to have this behavior?

**Comment by Enivex** [comment]
`#v(1fr)`

**Comment by cristobaltapia** [comment]
Thanks! That was exactly :)
